.github/CODEOWNERS

```diff
@@ -6,6 +6,10 @@
 # Default owners
 * @pandemicsyn @edgarrmondragon
 
+# CI
+.github/actions/ @WillDaSilva
+.github/workflows/ @WillDaSilva
```

Comment:
@WillDaSilva - Do you mind adding either a backup approver as secondary, and/or the `@meltano/engineering` team as a collective backup?